arithmetization/src/test/java/net/consensys/linea/zktracer/containers/ModuleOperationStackedSetTests.java

```diff
@@ -62,10 +62,52 @@ public void multiplePushPop() {
 
     chunks.enter();
     chunks.add(ONE_PLUS_ONE);
+    Assertions.assertEquals(2, chunks.size());
+    chunks.add(ONE_PLUS_TWO);
+    Assertions.assertEquals(3, chunks.size());
+    chunks.pop();
     Assertions.assertEquals(1, chunks.size());
+  }
+
+  @Test
+  public void deleteDuplicate() {
+    ModuleOperationStackedSet<AddOperation> chunks = new ModuleOperationStackedSet<>();
+    chunks.enter();
+    chunks.add(ONE_PLUS_ONE);
+    chunks.add(ONE_PLUS_ONE);
+    Assertions.assertEquals(1, chunks.size());
+
+    chunks.enter();
+    chunks.add(ONE_PLUS_ONE);
     chunks.add(ONE_PLUS_TWO);
+    Assertions.assertEquals(3, chunks.size());
+    chunks.lineCount();
     Assertions.assertEquals(2, chunks.size());
     chunks.pop();
     Assertions.assertEquals(1, chunks.size());
+
+    chunks.enter();
+    Assertions.assertEquals(1, chunks.size());
+    chunks.add(ONE_PLUS_ONE);
+    chunks.add(ONE_PLUS_TWO);
+    Assertions.assertEquals(3, chunks.size());
+    chunks.enter();
+    Assertions.assertEquals(2, chunks.size());
+  }
+
+  @Test
+  public void lineCount() {
+    ModuleOperationStackedSet<AddOperation> chunks = new ModuleOperationStackedSet<>();
+    chunks.enter();
+    chunks.add(ONE_PLUS_ONE);
+    chunks.add(ONE_PLUS_ONE);
+    Assertions.assertEquals(2, chunks.lineCount());
```

Comment:
Why do we have 2 here and not 1 ?

Reply:
I guess lineCount() on AddOperation returns 2 for each of these operations, basically ctMax = 1., correct ?